test.js

```diff
@@ -131,3 +133,66 @@ test('unist-util-is', function (t) {
 
   t.end()
 })
+
+test('unist-util-is properties', function (t) {
+  t.plan(4)
+  t.doesNotThrow(function () {
+    fc.assert(
+      fc.property(
+        fc.record({type: fc.string({minLength: 1})}),
```

Comment:
`unist-util-is` does not consider
```json
{
    "type": ""
}
```

to be a valid node

Reply:
I agree that `type` should be a non-empty string: https://github.com/syntax-tree/unist#node